app/src/views/SurgeCatalogue/index.tsx

```diff
@@ -162,11 +162,81 @@ export function Component() {
                     {strings.catalogueWater}
                 </NavigationTab>
                 <NavigationTab
-                    to="surgeCatalogueOther"
+                    to="surgeCatalogueOtherCivilMilitaryRelations"
                     parentRoute
                 >
                     <MoreOptionsIcon className={styles.icon} />
-                    {strings.catalogueOther}
+                    {strings.catalogueCMR}
+                </NavigationTab>
+                <NavigationTab
+                    to="surgeCatalogueOtherDisasterRiskReduction"
+                    parentRoute
+                >
+                    <MoreOptionsIcon className={styles.icon} />
```

Comment:
@szabozoltan69 is the choice of icon correct?

Reply:
Yes, the decision was to ask it from Mariam but until then we use this "Other" icon.